### PR TITLE
CB-20434 first refactored applied plugins to the new format and tried…

### PR DESCRIPTION
**Pull request by lnardai**
… to remove unnecessary plugins from libraries where spring should not be applied for sure. 

- Idbms client had spring test lib dependency so I've refactored the testcases since it's overkill
- Moved away from old classpath + apply plugin syntax to new one.

### CODE REVIEW

File: idbmms-connector/src/test/java/com/sequenceiq/cloudbreak/idbmms/config/IdbmmsConfigTest.java [hunk 1]
```diff
@@ -1,12 +1,11 @@
 package com.sequenceiq.cloudbreak.idbmms.config;
 
-import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertFalse;
```
Comment (bergerdenes) [inline idbmms-connector/src/test/java/com/sequenceiq/cloudbreak/idbmms/config/IdbmmsConfigTest.java:3]: please no junit4

File: idbmms-connector/src/test/java/com/sequenceiq/cloudbreak/idbmms/exception/IdbmmsOperationExceptionTest.java [hunk 2]
```diff
@@ -15,11 +15,11 @@ class IdbmmsOperationExceptionTest {
...
     static Object[][] getErrorStatusDataProvider() {
-        return new Object[][] {
+        return new Object[][]{
                 // testCaseName             withCause   code                    errorStatusExpected
-                { "no cause",               false,      null,                   IdbmmsOperationErrorStatus.UNSPECIFIED },
-                { "with cause null",        true,       null,                   IdbmmsOperationErrorStatus.UNSPECIFIED },
-                { "with cause NOT_FOUND",   true,       Status.Code.NOT_FOUND,  IdbmmsOperationErrorStatus.NOT_FOUND },
+                {"no cause", false, null, IdbmmsOperationErrorStatus.UNSPECIFIED},
+                {"with cause null", true, null, IdbmmsOperationErrorStatus.UNSPECIFIED},
+                {"with cause NOT_FOUND", true, Status.Code.NOT_FOUND, IdbmmsOperationErrorStatus.NOT_FOUND},
```
Comment (bergerdenes) [inline idbmms-connector/src/test/java/com/sequenceiq/cloudbreak/idbmms/exception/IdbmmsOperationExceptionTest.java:22]: How come that the @formatter:off is ignored? Never happened before

**Comment by bergerdenes** [comment]
Big like! 👍🏻

**Comment by keyki** [comment]
Closing the PR, please re-open if it's still relevant.

**Pull request closed without merging**